cds_dojson/marc21/fields/books/base.py

```diff
@@ -443,8 +463,11 @@ def alternative_identifiers(self, key, value):
         else:
             raise UnexpectedValue(subfield='2')
     if key == '035__':
+        if 'CERCER' in sub_a:
+            raise IgnoreKey('alternative_identifiers')
         sub_9 = clean_val('9', value, str, req=True)
-
+        if 'CERCER' in sub_9:
```

Comment:
nit: the obvious

Reply:
I'm sorry, could you elaborate on the obvious? I don't know what you mean